CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.7 (2018-12-13)
+
+- Added option to take action before modals are closed and prevent them from closing. [#7](https://github.com/blackbaud/skyux-modals/pull/7)
```

Comment:
Seems like we should specify that this new property is related to unsaved work in the modal. Or is there another use case for this? I'm thinking something along the lines of:

"Added option to prompt users before closing modals with unsaved work."

Would something like that be accurate?

Reply:
Btw, if I'm not following this property correctly please let me know. I have a short description in https://github.com/blackbaud/skyux2/issues/2138 to update the docs, but if it's not accurate or not nuanced enough, please let me know so that I can get the docs right.

Reply:
Oh yeah that is very outdated. Here let me make a PR instead. I don't think we need to directly call out unsavedWork by the way. This is now just a way to prevent modals from closing. Unsaved work may be a reason, but its not the only one